anvio/docs/artifacts/concatenated-gene-alignment-fasta.md

```diff
@@ -0,0 +1,14 @@
+This file **contains the alignment information for multiple genes across different organisms**.
+
+Basically, a single gene alignment compares a single gene's sequence across multiple organisms. For example, you could align some specific rRNA sequence across all of the organisms in your sample. This alignment highlights both mutations and insertions and deletions (indicated with dashes). 
+
+Clustal programs do a great job of visualizing this data, by color coding it. Here is an example from Anvi'o's pangenome display: 
+
+A lovely clustal-like alignment from the anvi'o pangenome display
```

Comment:
Is this a placeholder for an image? :)

Reply:
Yep! Andrea said she'd send me an example from the pangenome display soon, so I put that there as a placeholder for the time being.